src/WpfMath/CharBox.cs

```diff
@@ -22,31 +23,20 @@ public CharInfo Character
             private set;
         }
 
-        private GlyphRun GetGlyphRun(double scale, double x, double y)
+        internal GlyphRun GetGlyphRun(double scale, double x, double y)

```

Comment:
`GlyphRun` is WPF class. Please don't forget create issue to move this part to WPF renderer.

Reply:
- [ ] also, this particular line (changing of `private` to `internal`) seem unnecessary?

Thanks for your review!

Reply:
> this particular line (changing of `private` to `internal`) seems to be unnecessary?

Nope, it doesn't. This method is called from the test code.